src/script/util/util.ts

```diff
@@ -255,7 +255,8 @@ export const downloadFile = (url: string, fileName: string, mimeType?: string):
   }, 100);
 };
 
-export const createRandomUuid = (): string => UUID.genV4().toString();
+export const createRandomUuid = (version: 4 | 1 = 4): string =>
+  (version === 4 ? UUID.genV4() : UUID.genV1()).toString();
```

Comment:
why do we need v1 for this? I guess v4 would do nicely as well, isn't it?

Reply:
Backend does return 500 error when v4 is used (v4 does not contain timestamps anymore what I believe is the case).